src/main/java/graphql/nadel/dsl/NodeId.java

```diff
@@ -21,6 +22,10 @@ public static String getId(Node<?> node) {
         return assertNotNull(node.getAdditionalData().get(ID), () -> String.format("expected node %s to have an id", node));
     }
 
+    public static String getId(GraphQLFieldDefinition graphQLFieldDefinition) {
```

Comment:
@andimarek in response to your comments that I should use existing util methods for getting field Ids.

I added this new util method, is that ok? I am not using `assertNotNull` yet as some tests are failing if I start using it. I am checking how to fix the tests. I think the problem is that some tests have mock data without field id in the additional data inside of `FieldDefinition`. I assume that 
`getDefinition().getAdditionalData().get(ID)` should always be not null, is that correct?

Reply:
yes it should always be non null